tutorials/Quantile_Interval_And_Interval_Scores.ipynb

```diff
@@ -83,11 +92,13 @@
    "metadata": {},
    "source": [
     "In this example, we generate synthetic observation data and interval forecasts for temperature under two scenarios:\n",
-    "1. **Scenario 1**: There is a 50% chance that the temperature will be between 15 and 20 °C, with a lower quantile of 0.10 (indicating 10% cooler) and an upper quantile of 0.60 (indicating 40% warmer).\n",
-    "2. **Scenario 2**: There is a 50% chance that the temperature will be between 15 and 20 °C, with a lower quantile of 0.25 (indicating 25% cooler) and an upper quantile of 0.75 (indicating 25% warmer).\n",
+    "\n",
+    "**Scenario 1**: There is a 50% chance that the temperature will be between 15 and 20 °C, with a lower quantile of 0.10 (indicating 10% cooler) and an upper quantile of 0.60 (indicating 40% warmer).\n",
+    "\n",
+    "**Scenario 2**: There is a 50% chance that the temperature will be between 15 and 20 °C, with a lower quantile of 0.25 (indicating 25% cooler) and an upper quantile of 0.75 (indicating 25% warmer).\n",
```

Comment:
I was reflecting further on this. I think the wording "indicating x% cooler/warmer" could be confusing to readers. I think it would be preferable if these were phrased differently to be clearer to readers.